### DISCUSSION TITLE
email's

### DISCUSSION THREAD

**Discussion by interactive-everything** [community/community#21222] [Other Feature Feedback, Questions, & Ideas] [answered]
why do i get so many email's please fix this.

**Comment by ghost** [comment]
At the end of the e-mail there is this word "unsubscribe"
You can cancel the presence of any spam e-mails."
![IMG_20220725_060450](https://user-images.githubusercontent.com/108471229/180696791-b1f8ec3a-0c20-4973-a63b-d85259a75477.jpg)

**Reply by H62rahnama** [reply]
h62rahnama

**Comment by ihnatenkoo** [comment, marked as answer]
**Go to the settings and then choose Notifications, here you can set up notifications**

![1231](https://user-images.githubusercontent.com/89586345/180742361-a27a983d-5cfc-4f02-8fed-4591574954c7.png)
